targets/unity-v2/Testing/Editor/PlayFabPackager.cs

```diff
@@ -16,6 +56,8 @@ public static class PlayFabPackager
             "assets/Testing/scenes/testscene.unity"
         };
 
+    private static readonly string TestPackageName = "com.playfab.service";
```

Comment:
nitpick: Can we place members at the top of the class? 

I can see the use of keeping variables closer to where they are used, but it helps for general code navigation to organize members at the top of the file for use anywhere else in the file. 

Thoughts?

Reply:
Isn't it at the top already?  The only thing ahead of it is another static readonly.